test/data/test_mapper.py

```diff
@@ -41,6 +47,32 @@ def test_categorical_tensor_mapper():
     pd.testing.assert_series_equal(out, pd.Series(['A', 'B', None, None, 'B']))
 
 
+def test_timestamp_tensor_mapper():
+    start_date = datetime(2000, 1, 1)
+    end_date = datetime(2023, 1, 1)
+    num_rows = 10
+    format = '%Y-%m-%d %H:%M:%S'
+    arr = [
+        _random_timestamp(start_date, end_date, format)
+        for _ in range(num_rows)
+    ]
+    arr[0] = np.nan
+    ser = pd.Series(arr)
+    stype = torch_frame.timestamp
+
+    year_range = compute_col_stats(ser, stype,
+                                   time_format=format)[StatType.YEAR_RANGE]
+
+    mapper = TimestampTensorMapper(format=format, year_range=year_range)
+
+    out = mapper.forward(ser)
+    assert out.shape == (num_rows, 1, 7)
+    assert torch.isnan(out[0, :, :]).all()
+    assert out[1:, :, :].dtype == torch.float32
+    is_between_0_and_1 = (out[1:, :, :] >= 0) & (out[1:, :, :] < 1)
+    assert is_between_0_and_1.all()
```

Comment:
I think we can generate some specified timestamp (not random), and test if we can read off the correct timestamp from the PyTorch Tensor.